website/docs/integrations/hive/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 2
+title: Installation
```

Comment:
There should be information about supported Hive versions, either in installation page or in integration description page

Reply:
info added